nibabel/tests/test_testing.py

```diff
@@ -142,3 +142,24 @@ def f():
         with suppress_warnings():
             raise ValueError('An error')
     assert_raises(ValueError, f)
+
+
+def test_assert_re_in():
+    assert_re_in(".*", "")
+    assert_re_in(".*", ["any"])
+
+    # should do match not search
+    assert_re_in("ab", "abc")
+    assert_raises(AssertionError, assert_re_in, "ab", "cab")
+    assert_raises(AssertionError, assert_re_in, "ab$", "abc")
+
+    # Sufficient to have one entry matching
+    assert_re_in("ab", ["", "abc", "laskdjf"])
+    assert_raises(AssertionError, assert_re_in, "ab$", ["ddd", ""])
+
+    # Tuples should be ok too
+    assert_re_in("ab", ("", "abc", "laskdjf"))
+    assert_raises(AssertionError, assert_re_in, "ab$", ("ddd", ""))
+
+    # shouldn't "match" the emty list
```

Comment:
thanks, will do/fix

someone might want/will do the option whenever necessity comes ;-)